src/main/java/org/terasology/gookeeper/system/GooeySystem.java

```diff
@@ -65,6 +65,13 @@
 
 @RegisterSystem
 public class GooeySystem extends BaseComponentSystem implements UpdateSubscriberSystem {
+
+    private static final int NUM_OF_ENTITIES_ALLOWED = 10;
+    private static int currentNumOfEntities = 0;
+    private static final float MAX_DISTANCE_FROM_PLAYER = 60f;
+    private static final Logger logger = LoggerFactory.getLogger(GooeySystem.class);
```

Comment:
same here, also I think the static int (without final) should be the last in line

```suggestion
    private static final Logger logger = LoggerFactory.getLogger(GooeySystem.class);
    private static final int NUM_OF_ENTITIES_ALLOWED = 10;
    private static final float MAX_DISTANCE_FROM_PLAYER = 60f;
    private static int currentNumOfEntities = 0;
```